docs/axes/radial/linear.md

```diff
@@ -104,6 +104,7 @@ The following options are used to configure the point labels that are shown on t
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
+| `display` | `boolean` | `true` | if true, point labels are shown.
```

Comment:
Do we still need the sentence just before this table?

> Note that these options only apply if `pointLabels.display` is true.

Reply:
Agree that this note is not needed. Done.